pkg/danm/danm.go

```diff
@@ -183,13 +191,115 @@ func extractConnections(args *cniArgs) error {
   return nil
 }
 
+func getResourcePrefix(args *cniArgs, resourceType string)(string,error){
+  confArgs, err := loadNetConf(args.stdIn)
+  if err != nil {
+    return "", errors.New("cannot load CNI NetConf due to error:" + err.Error())
+  }
+  k8sClient, err := createK8sClient(confArgs.Kubeconfig)
+  if err != nil {
+    return "", errors.New("cannot create K8s REST client due to error:" + err.Error())
+  }
+  configmap, err := k8sClient.CoreV1().ConfigMaps(string("kube-system")).Get(string("resource-prefix-map"), meta_v1.GetOptions{})
```

Comment:
what if ConfigMap does not exist, and the parameter is not even configured?
I guess your code fails, which makes the implementation backward incompatible

the new DP support shall be optional. if parameter is not present, or ConfigMap does not exist, we should simply ignore

Reply:
ConfigMap has been removed.